src/ccontrol/UserQuerySelect.cc

```diff
@@ -317,6 +326,280 @@ void UserQuerySelect::submit() {
+        auto& chunkSpec = *i;
+
+        // Make the JobQuery now
+        QSERV_LOGCONTEXT_QUERY(_qMetaQueryId);
+
+        qproc::ChunkQuerySpec::Ptr cs;  //&&&diff old one did this in lambda
+        {
+            std::lock_guard<std::mutex> lock(chunksMtx);
+            cs = _qSession->buildChunkQuerySpec(queryTemplates, chunkSpec);
+            chunks.push_back(cs->chunkId);
+        }
+        std::string chunkResultName = ttn.make(cs->chunkId);
+
+        LOGS(_log, LOG_LVL_WARN, "&&& UserQuerySelect::submitNew d2");
+        // This should only need to be set once as all jobs should have the same database name.
+        //&&& this probably has to do with locating xrootd resources, need to check. ???
+        if (cs->db != dbName) {
+            LOGS(_log, LOG_LVL_WARN, "&&& dbName change from " << dbName << " to " << cs->db);
+            if (dbNameSet) {
+                throw util::Bug(ERR_LOC, "Multiple database names in UBerJob");
+            }
+            dbName = cs->db;
+            dbNameSet = true;
+        }
+
+        //&&& TODO:UJ for UberJobs, cmr and MerginHandler wont be needed unless the uber job fails. could
+        // probably save some time.
+        //&&& std::shared_ptr<ChunkMsgReceiver> cmr = ChunkMsgReceiver::newInstance(cs->chunkId,
+        //_messageStore);
+        LOGS(_log, LOG_LVL_WARN, "&&& UserQuerySelect::submitNew d3");
+        ResourceUnit ru;
+        ru.setAsDbChunk(cs->db, cs->chunkId);
+        qdisp::JobDescription::Ptr jobDesc = qdisp::JobDescription::create(
+                _qMetaCzarId, _executive->getId(), sequence, ru,
+                //&&&std::make_shared<MergingHandler>(cmr, _infileMerger, chunkResultName),
+                std::make_shared<MergingHandler>(_infileMerger, chunkResultName), taskMsgFactory, cs,
+                chunkResultName);
+        auto job = _executive->add(jobDesc);
+
+        LOGS(_log, LOG_LVL_WARN, "&&& UserQuerySelect::submitNew d4");
+        if (!uberJobsEnabled) {
+            LOGS(_log, LOG_LVL_WARN, "&&& UserQuerySelect::submitNew d4a");
+            // references in captures cause races
+            auto funcBuildJob = [this, job{move(job)}](util::CmdData*) {
+                QSERV_LOGCONTEXT_QUERY(_qMetaQueryId);
+                //&&& job->runJob();
+                LOGS(_log, LOG_LVL_WARN, "&&& UserQuerySelect::submitNew lambda a");
+                _executive->runJobQuery(job);
+                LOGS(_log, LOG_LVL_WARN, "&&& UserQuerySelect::submitNew lambda b");
+            };
+            auto cmd = std::make_shared<qdisp::PriorityCommand>(funcBuildJob);
+            LOGS(_log, LOG_LVL_WARN, "&&& UserQuerySelect::submitNew d4b");
+            _executive->queueJobStart(cmd);
+            LOGS(_log, LOG_LVL_WARN, "&&& UserQuerySelect::submitNew d4c");
+        }
+        ++sequence;
+    }
+
+    LOGS(_log, LOG_LVL_WARN, "&&& UserQuerySelect::submitNew e");
+    if (uberJobsEnabled || true) {
```

Comment:
This is where it is sending the http/json version of the request to the worker. At this time, the worker parses this, sends a brief message back to the czar, and then throws it away.